processing/physical_range.py

```diff
@@ -312,93 +310,58 @@ def make_plot(self, x_data, y_data, var_obj, year, thresholds=True):
+        fig.suptitle(f'Physical Range of {var_obj.name} throughout {y_name}', fontsize=self.plot_config.plot_suptitle_fontsize)
+
+        self.make_plot(ax1, time_data, annual_data, var_obj, year, thresholds=True)
+        self.make_plot(ax2, time_data, annual_data, var_obj, year, thresholds=False)
+        
         lab_warn = f'Expected Range ({var_obj.min_lim}-{var_obj.max_lim})'
         margin_percent = str(var_obj.margin * 100)
-        lab_err = (f'Expected Range +/- {margin_percent}% '
-                   f'({var_obj.error_min}-{var_obj.error_max})')
+        lab_err = (f'Expected Range +/- {margin_percent}% ({var_obj.error_min}-{var_obj.error_max})')
 
         legend_info = [('orange', lab_warn), ('r', lab_err)]
-        handles = [mlines.Line2D([], [], color=c, label=l) for c, l in
-                   legend_info]  # convert legend info to Patches
+        handles = [mlines.Line2D([], [], color=c, label=l) for c, l in legend_info]
         labels = [l for c, l in legend_info]
 
-        # Add labels and colors for point styling
-        handles += [mlines.Line2D([], [], color='None', marker='o',
-                                  markerfacecolor='0.75', markersize=10,
-                                  label='data'),
-                    mlines.Line2D([], [], color='None', marker='o',
-                                  markerfacecolor='None',
-                                  markeredgecolor='orange',
-                                  markersize=10,
-                                  label=('data outside the expected '
-                                         'physical range')),
-                    mlines.Line2D([], [], color='None', marker='o',
-                                  markerfacecolor='None', markeredgecolor='r',
-                                  markersize=10,
-                                  label=(
-                                      'data outside the expected physical '
-                                      f'range +/- {margin_percent}%'))]
-        labels += ['data', 'data outside the expected physical range',
-                   'data outside the expected physical range']
-        plt.figlegend(handles, labels, loc='lower center', ncol=2)
-
-        fig_name = self.fig_name_fmt.format(
-            y=var_obj.name, year=year, s=self.site_id, p=self.process_id)
+        handles += [mlines.Line2D([], [], color='None', marker='o', markerfacecolor='0.75', markersize=10, label='data'),
+                    mlines.Line2D([], [], color='None', marker='o', markerfacecolor='None', markeredgecolor='orange', markersize=10, label='data outside the expected physical range'),
+                    mlines.Line2D([], [], color='None', marker='o', markerfacecolor='None', markeredgecolor='r', markersize=10, label=f'data outside the expected physical range +/- {margin_percent}%')]
+        labels += ['data', 'data outside the expected physical range', f'data outside the expected physical range +/- {margin_percent}%']

```

Comment:
These lines look largely the same as those above, just formatted differently. True?

Reply:
Yeah, I just changed the formatting of a few lines but the code's the same